request_test.go

```diff
@@ -2,6 +2,7 @@ package sarama
 
 import (
 	"bytes"
+	"reflect"
```

Comment:
My understanding was that importing `reflect` at any point required the go compiler to include a bunch more information in the final library/binary, but I'm not actually sure if that is true for testing as well